tests/cpp_tests/test_mvs_threshold_search.cpp

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) 2021 Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023 Microsoft Corporation. All rights reserved.
```

I think this should be 2023, since this will be a new file added this year.